### DISCUSSION TITLE
showToggleColumns not responding

### DISCUSSION THREAD

**Discussion by muisawe** [Power-Components/livewire-powergrid#498] [Show and tell]
## Information

#### Have you searched through other issues to see if your problem is already reported or has been fixed?
- [X ] Yes - I did not find it.
- [ ] No

#### Did you read the [documentation](https://livewire-powergrid.com/)?
- [ X] Yes - I did not find it.
- [ ] No

#### Have you tried to publish the views?

- [ X] Yes - I didn't work.
- [ ] No, this error is not related to views.

#### Is there an error in the console?
- [ ] Yes - I'll add a screenshot or report it below.
- [ X] No

### Software Version

| Software  | Version (exactly) |
|-----------|-------------------|
| PowerGrid |     v3.2.1               |
| Laravel   |         v9.19           |
| Livewire  |                v2.10.6   |
| Alpine JS  |        3.x.x           |

### Theme
- [ ] Tailwind 2.x
- [ ] Tailwind 3.x
  - [ ] With tailwind/forms
- [X ] Bootstrap - 5.1 :        

---
### What happened?

I did tried to use  showToggleColumns the panel was appeared but it's not responding 

### To Reproduce...

select the columns toggle button and try to select any of hide/show elements 'depend in your model'

### Suggestions

No. 

## Extra information

**Screenshots**
<img width="175" alt="Screen Shot 2022-07-08 at 1 04 28 AM" src="https://user-images.githubusercontent.com/26344769/177880028-e6e8dcaa-f6d8-4e0b-864f-61976359fce6.png">

**Code snippet**

```php 
<?php
  public function setUp(): array
    {
        $this->showCheckBox();
        $this->persist(['columns', 'filters']);

        return [
            Exportable::make('Adminstrations')
                ->striped('27bcfd')
                ->type(Exportable::TYPE_XLS, Exportable::TYPE_CSV),
            Header::make()
                ->showToggleColumns()
                ->showSearchInput(),
            Footer::make()
                ->showPerPage()
                ->showRecordCount(),
        ];
    }
//...
```

**Comment by luanfreitasdev** [comment]
Hi!

I believe this happened because of the bootstrap version, as powergrid only supports 5.0, maybe some css class has changed. I'll convert this into Discussion, I accept a PR to adapt in this case if that's all that stopped working.

Thanks
